package.json

```diff
@@ -52,8 +52,8 @@
     "prettier-plugin-packagejson": "^2.2.11",
     "rimraf": "^3.0.2",
     "ts-jest": "^26.3.0",
-    "typedoc": "^0.22.15",
-    "typescript": "^4.2.4"
+    "typedoc": "^0.23.10",
+    "typescript": "~4.7.4"
```

Comment:
Why `~` instead of `ˆ`?

Reply:
TypeScript doesn't follow SemVer; minor changes are breaking